src/lambda_alerting.py

```diff
@@ -39,48 +43,48 @@ def write_event_to_timestream(records):
     logger.info(result)
 
 
-def settings_stub_get_monitored_env_name(account, region):
-    """
-    A stub function to return a monitored environment name based on account and region.
-    To be replaced when Settings Component is ready
+def read_settings(s3_bucket_name: str) -> Settings:
+    """Reads settings from the dedicated S3 bucket
 
     Args:
-        account (str): AWS account ID.
-        region (str): AWS region name.
+        s3_bucket_name (str): Name of the Settings S3 bucket
 
     Returns:
-        str: A string representing the name of the monitored environment.
+        Settings: Setting container
     """
-    return "monitored_env_from_stub"
+    settings_path = f"s3://{s3_bucket_name}/settings/"
```

Comment:
I believe, it's better to pass the full S3 path as env variable into lambda from CDK.
instead of just passing bucketname